conda/environment-test-minimal-deps.yml

```diff
@@ -55,6 +55,7 @@ dependencies:
   - pip:
     - mypy >=1.10
     # docs
+    # omit bokeh_sampledata
```

Comment:
It jsut occurs to me, why are we installing anything docs related in the "minimal" env? Perhaps all of these can be removed?